src/System.Private.CoreLib/shared/System/Diagnostics/DebugProvider.Unix.cs

```diff
@@ -10,8 +10,14 @@ public partial class DebugProvider
     {
         private static readonly bool s_shouldWriteToStdErr = Environment.GetEnvironmentVariable("COMPlus_DebugWriteToStdErr") == "1";
 
-        public virtual void ShowDialog(string stackTrace, string message, string detailMessage, string errorSource)
+        public static void ShowDialog(string stackTrace, string message, string detailMessage, string errorSource)
```

Comment:
This does not actually show any dialog in .NET Core today. Should it be rather called something else?

Reply:
FailCore perhaps?